integration-tests/cli/resource-deleter.it.js

```diff
@@ -0,0 +1,345 @@
+
+  describe.each(Object.keys(resources).reverse())(
+    'should delete resource',
+    resource => {
+      beforeEach(() => {
+        const options = {
+          apiConfig,
+          resource,
+          logger,
+        }
+        resourceDeleter = new ResourceDeleter(options)
+      })
+
+      it(`${resource} deleted`, async () => {
+        // Check that resource exists
+        const payload = await getResource(resource)
+        expect(payload.body.results.length).toBeGreaterThanOrEqual(1)
+
+        // Delete resource
+        await resourceDeleter.run()
+
+        // Check that resource is deleted
+        const newPayload = await getResource(resource)
+        expect(newPayload.body.results).toHaveLength(0)
+      }, 25000)
+    }
+  )
+
+  describe('should delete a specific resource', () => {
+    const resource = 'channels'
+    beforeEach(() => {
+      const options = {
+        apiConfig,
+        resource,
+        logger,
+      }
+      resourceDeleter = new ResourceDeleter(options)
+      return createData(apiConfig, resource, [
+        {
+          key: 'singleChannel',
+          name: {
+            en: 'singleChannel',
+            de: 'singleChannel',
+          },
+        },
+        {
+          key: 'nextChannel',
+          name: {
+            en: 'nextChannel',
+            de: 'nextChannel',
+          },
+        },
+      ])
+    })
+
+    it(`The specified ${resource} deleted`, async () => {
+      const payload = await getResource(resource)
+      expect(payload.body.results.length).toBeGreaterThanOrEqual(1)
+      await resourceDeleter.run()
+      const newPayload = await getResource(resource)
+      expect(newPayload.body.results).toHaveLength(0)
+    })
+  }, 15000)
+
+  describe('should delete a specific resource with predicate', () => {
+    const resource = 'customerGroups'
+    beforeEach(() => {
+      const options = {
+        apiConfig,
+        resource,
+        logger,
+        predicate: 'key="CGKey2"',
+      }
+      resourceDeleter = new ResourceDeleter(options)
+      return createData(apiConfig, resource, [
+        {
+          key: 'CGKey1',
+          groupName: 'SampleCGName1',
+        },
+        {
+          key: 'CGKey2',
+          groupName: 'SampleCGName2',
+        },
+        {
+          key: 'CGKey3',
+          groupName: 'SampleCGName3',
+        },
+      ])
+    })
+
+    it(`The specified ${resource} with predicate deleted`, async () => {
+      const payload = await getResource(resource)
+      expect(payload.body.results).toHaveLength(3)
+      await resourceDeleter.run()
+      const newPayload = await getResource(resource)
+      expect(newPayload.body.results).toHaveLength(2)
+    }, 15000)
+  })
+
+  describe('should delete categories with its grandchildren', () => {
+    const resource = 'categories'
+    beforeAll(async () => {
+      const options = {
+        apiConfig,
+        resource,
+        logger,
+      }
+
+      const parentCategories = [
```

Comment:
Moving it to the file will be a problem when trying to create categories like `parentCategories`, `childCategories` etc because bluebird promise is used to import from the file.
Importing only some part of the data is not possible.
```
await Promise.each(Object.keys(resources), name => {
      return createData(apiConfig, name, resources[name])
    })
```

Reply:
Ahh, makes sense, thanks!